dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -399,6 +399,7 @@ public Object createAnyDataSourceOrDriver(Properties props) throws SQLException
      * <li>javax.sql.XADataSource
      * <li>javax.sql.ConnectionPoolDataSource
      * <li>javax.sql.DataSource
+     * <li>java.sql.Driver
```

Comment:
It is incorrect to place Driver in this ordered list of data source precedence due to our reliance on the URL attribute

Reply:
Same as above, I was adding the entry to match the other list but I'll remove.